lib/cea/cea708_service.js

```diff
@@ -0,0 +1,677 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.cea.Cea708Service');
+
+/**
+ * CEA-708 closed captions service as defined by CEA-708-E. A decoder can own up
+ * to 63 services. Each service owns eight windows.
+ */
+shaka.cea.Cea708Service = class {
+  /**
+   * @param {!number} serviceNumber
+   */
+  constructor(serviceNumber) {
+    /**
+     * Number for this specific service (1 - 63).
+     * @private {!number}
+     */
+    this.serviceNumber_ = serviceNumber;
+
+    /**
+     * Eight Cea708 Windows, as defined by the spec.
+     * @private {!Array<?shaka.cea.Cea708Window>}
```

Comment:
You need to `goog.require(shaka.cea.Cea708Window)` first.

We are working toward a compiler upgrade that will more aggressively enforce this, but we've got some ~100 other instances of missing `require()`s, so it will take some time.  In the mean time, check over your PR and try to clean up any you find.  That will make our eventual upgrade easier.  Thanks!